api/helpers.go

```diff
@@ -140,13 +140,17 @@ func (st *DCOSTools) GetUnitNames() (units []string, err error) {
 func (st *DCOSTools) GetJournalOutput(unit string) (string, error) {
 	matches := []reader.JournalEntryMatch{
 		{
+			Field: "_SYSTEMD_UNIT",
```

Comment:
I just realized that the `Value` part of the struct here is supplied as a method argument.  So yeah, possibly we would need to just retain the `Field` parts here.